src/indexClinicalData/index.ts

```diff
@@ -0,0 +1,70 @@
+import transformToEsDonor from "./transformToEsDonor";
+import programDonorStream from "./programDonorStream";
+import { toEsBulkIndexActions } from "elasticsearch";
+import { STREAM_CHUNK_SIZE } from "config";
+import { Client } from "@elastic/elasticsearch";
+import logger from "logger";
+import { EsDonorDocument, EsHit } from "./types";
+import esb from "elastic-builder";
+
+export default async (
+  programShortName: string,
+  targetIndexName: string,
+  esClient: Client
+) => {
+  const donorStream = programDonorStream(programShortName, {
+    chunkSize: STREAM_CHUNK_SIZE,
+  });
+  let chunksCount = 0;
+  for await (const chunk of donorStream) {
+    const timer = `streaming ${
+      chunk.length
+    } donor(s) from chunk #${chunksCount++} of program ${programShortName}`;
+    logger.profile(timer);
+
+    const donorIds = chunk.map((donor) => `DO${donor.donorId}`);
+
+    const esQuery = esb
+      .requestBodySearch()
+      .query(esb.termsQuery("donorId", donorIds));
+
+    const esHits: Array<EsHit> = await esClient
+      .search({
+        // providing an index results in inablity to detect preixsting donors
+        //
+        // index: targetIndexName,
```

Comment:
I think this is because when we ask rollcall for a new index, we didn't told it to clone the old one :P should be a very small update to get that working ;)